site/src/components/WorkspaceDeletion/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from "./DeletionStat"
+export * from "./DeletionBadge"
+export * from "./DeletionText"
+export * from "./DeletionBanner"
```

Comment:
Do we really need this? Why not access those components directly?

Reply:
Good q!
1) Just a shorter directory to type out when importing, e.g. `components/WorkspaceDeletion` instead of `components/WorkspaceDeletion/XYZ` - I think it looks a little cleaner. This becomes a more important pattern if you have additional nesting.
2) I can also import multiple related components with [1 line](https://github.com/coder/coder/pull/7588/files/8322b633ee07b7e98038102d9323e575f4e9cdcb#diff-026d98d7ef3991784204fab8460cfc0d13cae28631293cb424bb13309b8d5de5R13) which keeps our file length down
3) Often times, the barrel file provides a clue as to how the components are meant to be used. For example, in this case, I am exporting everything, which means all components are meant to be consumed. Great. Let's say I was building a table that had a specific widget component, broken apart in a separate file, but used in the table and colocated in the same directory. In the barrel I would export the table component, but not the widget. A user unfamiliar with the code base could glance at the barrel file and see exactly what was meant to be consumed externally, i.e. the table and not the widget.